database/src/main/resources/db/migration/V1_22__Add_tx_events.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS tx_events (
+    id                   SERIAL PRIMARY KEY,
+    block_height         INT NOT NULL,
+    tx_hash_id           INT NOT NULL,
+    tx_hash              VARCHAR(64) NOT NULL,
+    tx_message_id        TEXT NOT NULL,
+    tx_message_type_id   VARCHAR(128) NOT NULL,
+    event_type           VARCHAR(256) NOT NULL,
+    tx_message_hash      TEXT NULL
+);
```

Comment:
We should also add `gas` fees to this table as well to avoid duplicate work I'm doing in #65 for generating stats. I was going to combine the data from `tx_cache`,`tx_message` and `tx_message_type` into a similar table to produce gas fee stats #185.

colums to add:
`tx_cache.gas_used`
`tx_cache.gas_wanted` @crussell-figure do we need this as well?

Reply:
Here is the transaction event that we are pulling the data for from:  https://service-explorer.test.provenance.io/api/v2/txs/B7164C6AC107613832CEFEC16EAD8A2B0002765292416DF9E9F611D89D29C50E/json. 

I didn't see gas_used or gas_wanted inside of there?  Unless I am missing something?

Reply:
@crussell-figure What are your thoughts?

Reply:
I dont think the fee values are appropriate on the events table. @fkneeland-figure you had marked off the fee table/columns in the PR, but I dont see anything for them. Maybe we should let @egaxhaj-figure do that stuff. Otherwise, I think the tables as presented here are fine.

Reply:
👍